incremental-font-transfer/src/patch_group.rs

```diff
@@ -647,6 +678,12 @@ mod tests {
     const TABLE_1_FINAL_STATE: &[u8] = "hijkabcdeflmnohijkabcdeflmno\n".as_bytes();
     const TABLE_2_FINAL_STATE: &[u8] = "foobarbaz foobarbaz foobarbaz\n".as_bytes();
 
+    impl PatchUri {
+        fn new(uri: &str) -> Self {
+            Self(uri.to_string())
+        }
```

Comment:
doesn't really matter for public API but in cases where you need an owned type in a fn it's generally better to accept the owned type as the argument:

- the caller might already have the owned type
- it makes the fact that the argument is cloned explicit

(you don't need to change this, it just reminded me of this idea)